Week 01/id_601/LeetCode_283_601.java

```diff
@@ -0,0 +1,18 @@
+class Solution {
+    public void moveZeroes(int[] nums) {
+        int j = 0;
+        for (int i = 0; i<nums.length; i++){
```

Comment:
nit:  one space between parenthesis and brace.

Reply:
thx